Atomic/syscontainers.py

```diff
@@ -926,12 +935,12 @@ def _do_checkout(self, repo, name, img, upgrade, deployment, values, destination
                     config = json.loads(config_file.read())
                 except ValueError:
                     raise ValueError("Invalid config.json file in given remote location: {}.".format(destination_path))
-                config['root']['path'] = remote_rootfs
+                config['root']['path'] = rootfs
```

Comment:
`rootfs` will probably have the wrong value.  I suggest to return remote_rootfs from the `_get_remote_location` function and keep using it here and on line 943

Reply:
> rootfs will probably have the wrong value

hmm, AFAICS, it seems like when remote_path is specified, rootfs will default to remote root fs anyway right? And this code section also lies under the condition of ( remote_path = true)? 

The following code seems to be the only place that does the change to the rootfs? In this code section, the rootfs will be assigned to remote rootfs location. i.e: remote/rootfs (I could be wrong tho, so feel free to correct me if I happened to understand it wrong)
https://github.com/projectatomic/atomic/blob/master/Atomic/syscontainers.py#L835-L848
```
if extract_only:
            rootfs = destination
        elif remote_path:
            rootfs = os.path.join(remote_path, "rootfs")
        else:
            destination = self._canonicalize_location(destination)
            rootfs = os.path.join(destination, "rootfs")

        if remote_path:
            if not os.path.exists(destination):
                os.makedirs(destination)
        else:
            if not os.path.exists(rootfs):
os.makedirs(rootfs)
```

Reply:
sure, I am fine in the `remote_path` case to consider `rootfs` always under `remote_path` itself